.github/workflows/docker.yml

```diff
@@ -0,0 +1,96 @@
+name: Red Hat
+on:
+  pull_request:
+  push:
+    branches: [ main ]
+jobs:
+  centos_stream:
+    name:  ${{ matrix.dist }}-${{ matrix.compiler}}-${{ matrix.build_type }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        dist:
+          - stream9
+          - stream10
+        build_type:
+          - Debug
+          - Release
+        compiler:
+          - gcc
+          - clang
+    container: quay.io/centos/centos:${{ matrix.dist }}
+    steps:
+      - name: Install development tools
+        run: |
+          dnf install -y epel-release
+          /usr/bin/crb enable
+          dnf install -y \
+              clang \
+              gcc \
+              g++ \
+              gfortran \
+              cmake \
+              make \
+              cfitsio-devel \
+              file \
+              git
+
+      - name: checkout
+        uses: actions/checkout@v4
+        with:
+          fetch-tags: true
+
+      - name: Fetch tags from upstream repository
+        run: |
+          git config --global --add safe.directory $(pwd) || true
+          git fetch --tags https://github.com/spacetelescope/hstcal
```

Comment:
Didn't you already ask checkout to fetch tags above?

Reply:
When this isn't present the version data shows up as `unknown`